libs/sdk/src/lib/kinetic-sdk-internal.ts

```diff
@@ -116,24 +139,27 @@ export class KineticSdkInternal {
     if (!this.appConfig) {
       throw new Error(`AppConfig not initialized`)
     }
-    const { mint, feePayer, latestBlockhash, lastValidBlockHeight } = await this.prepareTransaction()
+    mint = mint || this.appConfig.mint.publicKey
+    const { mintPublicKey, mintFeePayer, latestBlockhash, lastValidBlockHeight } = await this.prepareTransaction({
+      mint,
+    })
 
     const tx = await serializeMakeTransferTransaction({
       amount,
+      appIndex: this.appConfig.app.index,
       destination,
-      mint,
-      owner,
       latestBlockhash,
-      feePayer,
-      appIndex: this.appConfig.app.index,
+      mintFeePayer,
+      mintPublicKey,
+      owner,
       type: type || TransactionType.None,
     })
 
     const request: MakeTransferRequest = {
       commitment: commitment || Commitment.Confirmed,
       environment: this.appConfig.environment.name,
       index: this.appConfig.app.index,
-      mint: this.appConfig.mint.symbol,
```

Comment:
We mixed and matched `symbol` and `publicKey` in the Rest API. We now only use the `publicKey` of the mint.